File: nivel-02/ex-01/par_impar.c
#include <stdio.h>
#include <assert.h>

int ehPar(int numero) {
    if (numero % 2 == 0) {
        return 1;
    } else {
        return 0;
    }
}

int main (){

    assert(ehPar(8) == 1);
    assert(ehPar(7) == 0);
    assert(ehPar(0) == 1);

    return 0;
}

File: nivel-02/ex-01/par_impar.js
var assert = require('assert');

var ehPar = function(numero) {
    if (numero % 2 == 0) {
        return true;
    } else {
        return false;
    }
}

try {
    assert.ok(ehPar(8));
    assert.ok(!ehPar(7));
    assert.ok(ehPar(0));
} catch(e) {
    console.log(e);
}